circle.yml

```diff
@@ -36,6 +36,14 @@ test:
         - ./gradlew testDebugUnitTest -Pandroid.useDexArchive=false 
         #- ./gradlew assembleDebug -Pandroid.useDexArchive=false:
         #    timeout: 900
+        # start the emulator
+        - emulator -avd circleci-android24 -no-window:
```

Comment:
I think we should use android22 since that's our target SDK. I'm not sure though...

```
# start the emulator
- mksdcard -l e 128M sdcard.img
- emulator -avd circleci-android22 -sdcard sdcard.img -no-window:
    background: true
    parallel: true
```